products/memgraph/upcoming/concepts/replication.md

```diff
@@ -0,0 +1,77 @@
+## Replication
+
+## High Level Context
+
+Replication is a method that ensures that multiple database instances are
+storing the same data. To enable replication, there must be at least two
+instances of Memgraph in a cluster. Each instance has one of either two roles:
+main or replica. The main instance is the instance that accepts writes to the
+database and replicates its state to the replicas. In a cluster, there can only
+be one main. There can be one or more replicas. None of the replicas will accept
+write queries, but they will always accept read queries (there is an exception
+to this rule and is described below). Replicas can also be configured to be
```

Comment:
There is no support for the replicas of replicas yet (under Limitations). I would remove it from here to avoid confusion.